js/containers/portal-dashboard/feedback/question-feedback-panel.tsx

```diff
@@ -62,6 +72,17 @@ class QuestionFeedbackPanel extends React.PureComponent<IProps> {
       </div>
     );
   }
+
+  private updateQuestionFeedbackSettings = () => {
+    const questions = this.props.activity.get("questions");
+    questions.forEach((question: any) => {
+      const questionId = question.get("id");
+      if (!this.props.settings.get("questionSettings")?.get(questionId)?.get("feedbackEnabled")) {
+        this.props.updateQuestionFeedbackSettings(questionId, { feedbackEnabled: true });
+      }
+      this.props.updateQuestionFeedbackSettings(questionId, { feedbackEnabled: true });
```

Comment:
Is line 83 needed? Looks like it negates the need for lines 80-82.

Reply:
Yup.  My mistake.  Thanks for catching this!

Reply:
Fixed.